openmc/data/photon.py

```diff
@@ -858,6 +913,81 @@ def _add_bremsstrahlung(self):
         self.bremsstrahlung['photon_energy'] = _BREMSSTRAHLUNG['photon_energy']
         self.bremsstrahlung.update(_BREMSSTRAHLUNG[self.atomic_number])
 
+    def _compute_heating(self):
+        r"""Compute heating cross sections (KERMA)
+
+           Photon energy is deposited as energy loss in three reactions:
+           incoherent scattering, pair production and photoelectric effect.
+           The point-wise heating cross section is calculated as:
+
+           .. math::
+               \sigma_{Hx}(E) &= (E - \overline{E}_x(E)) \cdot \sigma_x(E), x \in \left\{I, PP, PE \right\}
+
+               \overline{E}_I(E) &= \frac {\int E' \sigma_I (E,E',\mu) d\mu} {\int \sigma_I (E,E',\mu) d\mu}
+
+               \overline{E}_{PP} &= 2 m_e c^2 = 1.022 \times 10^6 eV
+
+               \overline{E}_{PE} &= E(\text{fluorescent photons})
+
+           The differential cross section representation for incoherent
+           scattering can be found in the theory manual.
+
+        """
+
+        # Determine a union energy grid
+        energy = np.array([])
+        for mt in (504, 515, 517, 522):
+            if mt not in self:
+                warn('Cross sections for MT={} do not exist. The total heating '
```

Comment:
Did you run into data where this actually happens?

Reply:
No. This can be removed safely.